app/src/main/java/com/edulectronics/tinycircuit/Views/Adapters/ListViewAdapter.java

```diff
@@ -0,0 +1,93 @@
+package com.edulectronics.tinycircuit.Views.Adapters;
+
+import android.content.Context;
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageButton;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.edulectronics.tinycircuit.Models.Factories.ScenarioFactory;
+import com.edulectronics.tinycircuit.Models.Scenarios.IScenario;
+import com.edulectronics.tinycircuit.R;
+import com.edulectronics.tinycircuit.Views.CircuitActivity;
+import com.edulectronics.tinycircuit.Views.ExerciseMenuActivity;
+
+/**
+ * Created by Wilmer on 18-12-2016.
+ */
+
+public class ListViewAdapter extends BaseAdapter {
```

Comment:
Naam sugerreert dat dit heel generiek is maar op dit moment is deze klasse alleen bruikbaar voor een hoofdmenu. Naam veranderen of implementatie veranderen